site/content/samples/playwright/account-settings.js

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from '@playwright/test'
+
+test('account settings', async ({ page }) => {
+  await page.goto('https://danube-web.shop/')
+
+  await page.click('#login')
```

Comment:
@modern-sapien this is a general comment for all scripts: let's stick with the newest locator API and keep it consistent across the board. See [this page](https://playwright.dev/docs/locators) for all the details.

Reply:
Okay cool cool. I'll run back through files I changed and addressed this.

Reply:
some of the locator APIs do no address IDs, etc. 

There is no getByID or getByRole("input") that I could find, so some of these will stay as locators.

Reply:
@modern-sapien this is still the old API, please use `page.locator`